Realm/Tests/RealmTests.mm

```diff
@@ -1972,7 +1972,7 @@ - (void)testThawDifferentThread {
     }];
 }
 
--(void)testThawPreviousVersion {
+- (void)testThawPreviousVersion {
```

Comment:
Formatting.

Reply:
Which part?

Reply:
Nevermind, you have the correct formatting.